Source/Scene/Zones/BaseZone.tscn

```diff
@@ -531,12 +532,6 @@ scale = Vector2( -1, 1 )
 type = 1
 springDirection = 3
 
-[node name="Bumper" parent="Entity" instance=ExtResource( 13 )]
-position = Vector2( 6800, 1088 )
-
-[node name="Bumper2" parent="Entity" instance=ExtResource( 13 )]
-position = Vector2( 6896, 1088 )
-
```

Comment:
I moved these bumper gimmicks so that they could be positioned more neatly to take the place of the little platforms that Ice Cap Zone placed the original trampoline gimmick beside. It was important to reorder them so that they would be drawn *above* the terminal rings of the trampoline.